e2e/fixtures/schemas/result.json

```
@@ -0,0 +1,194 @@
+                    "query_id": {
+                        "type": "string",
+                        "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-4{1}[a-f0-9]{3}-[89ab]{1}[a-f0-9]{3}-[a-f0-9]{12}$"
+                    },
+                    "query_url": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "severity": {
+                        "type": "string",
+                        "enum": ["HIGH", "MEDIUM", "LOW", "INFO"]
+                    },
+                    "platform": {
+                        "type": "string",
+                        "enum": ["Ansible", "CloudFormation", "Dockerfile", "Kubernetes", "OpenAPI", "Terraform"]
+                    },
+                    "files": {
+                        "type": "array",
+                        "items": {
+                            "type": "object",
+                            "required": [
+                                "file_name",
+                                "similarity_id",
+                                "line",
+                                "issue_type",
+                                "search_key",
+                                "search_value",
+                                "expected_value",
+                                "actual_value",
+                                "value"
+                            ],
+                            "properties": {
+                                "file_name": {
+                                    "type": "string",
+                                    "minLength": 1
+                                },
+                                "similarity_id": {
+                                    "type": "string",
+                                    "pattern": "^[A-Fa-f0-9]{64}$"
+                                },
+                                "line": {
+                                    "type": "integer",
+                                    "minimum": 1
+                                },
+                                "issue_type": {
+                                    "type": "string",
+                                    "enum": ["IncorrectValue", "MissingAttribute", "RedundantAttribute"]
+                                },
+                                "search_key": {
+                                    "type": "string",
+                                    "minLength": 0
+                                },
+                                "search_value": {
+                                    "type": "string"
+                                },
+                                "expected_value": {
+                                    "type": "string",
+                                    "minLength": 0
+                                },
+                                "actual_value": {
+                                    "type": "string",
+                                    "minLength": 0
+                                },
```

I think `minLength` must be greater than 0, because it is a text informing the user about what is expected and what KICS just found...
```suggestion
                                "expected_value": {
                                    "type": "string",
                                    "minLength": 1
                                },
                                "actual_value": {
                                    "type": "string",
                                    "minLength": 1
                                },
```